[2023-02-20T15:45:24.778+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: predictit_dag.is_api_available manual__2023-02-20T05:38:00.897494+00:00 [queued]>
[2023-02-20T15:45:24.783+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: predictit_dag.is_api_available manual__2023-02-20T05:38:00.897494+00:00 [queued]>
[2023-02-20T15:45:24.784+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-02-20T15:45:24.784+0000] {taskinstance.py:1280} INFO - Starting attempt 2 of 2
[2023-02-20T15:45:24.784+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-02-20T15:45:24.792+0000] {taskinstance.py:1300} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-02-20 05:38:00.897494+00:00
[2023-02-20T15:45:24.795+0000] {standard_task_runner.py:55} INFO - Started process 1362 to run task
[2023-02-20T15:45:24.796+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'predictit_dag', 'is_api_available', 'manual__2023-02-20T05:38:00.897494+00:00', '--job-id', '563', '--raw', '--subdir', 'DAGS_FOLDER/predictit_dag.py', '--cfg-path', '/tmp/tmppjnbp9d7']
[2023-02-20T15:45:24.797+0000] {standard_task_runner.py:83} INFO - Job 563: Subtask is_api_available
[2023-02-20T15:45:24.827+0000] {task_command.py:388} INFO - Running <TaskInstance: predictit_dag.is_api_available manual__2023-02-20T05:38:00.897494+00:00 [running]> on host 9e0dc30811b3
[2023-02-20T15:45:24.948+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=kyunlee@yahoo.com
AIRFLOW_CTX_DAG_OWNER=kyun
AIRFLOW_CTX_DAG_ID=predictit_dag
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-02-20T05:38:00.897494+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-20T05:38:00.897494+00:00
[2023-02-20T15:45:24.949+0000] {http.py:122} INFO - Poking: all/
[2023-02-20T15:45:24.953+0000] {base.py:73} INFO - Using connection ID 'predictit_api' for task execution.
[2023-02-20T15:45:25.455+0000] {base.py:228} INFO - Success criteria met. Exiting.
[2023-02-20T15:45:25.461+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=predictit_dag, task_id=is_api_available, execution_date=20230220T053800, start_date=20230220T154524, end_date=20230220T154525
[2023-02-20T15:45:25.489+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-02-20T15:45:25.506+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
